src/haz3lweb/Export.re

```diff
@@ -49,6 +54,7 @@ let import_all = (~import_log: string => unit, data, ~specs) => {
         settings: all_f22.settings,
         scratch: all_f22.scratch,
         documentation: "",
+        tutorial: all_f22.tutorial,
```

Comment:
let's rename `all_f22` to `all_public` or something like that, since it isn't about F22 semester anymore :)